osu.Game/Overlays/BeatmapListing/BeatmapSearchGeneralFilterRow.cs

```diff
@@ -135,14 +141,28 @@ protected override void LoadComplete()
             {
                 base.LoadComplete();
 
+                config.BindWith(OsuSetting.BeatmapListingFeaturedArtistFilter, Active);
                 disclaimerShown = sessionStatics.GetBindable<bool>(Static.FeaturedArtistDisclaimerShownOnce);
+
+                // no need to show the disclaimer if the user already had it toggled off in config.
+                if (!Active.Value)
+                    disclaimerShown.Value = true;
+
+                if (game.HideUnlicensedContent)
+                {
+                    Enabled.Value = false;
+                    Active.Disabled = true;
+                }
```

Comment:
Should maybe show a tooltip why it's disabled. Something like `Toggling this filter is disabled in this platform`.

Reply:
Sure.

Reply:
https://github.com/ppy/osu/pull/31238/commits/6b635d588f16af12bde4340640aee476197795fd